dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/package-info.java

```diff
@@ -1,16 +1,15 @@
 /*******************************************************************************
- * Copyright (c) 2010 IBM Corporation and others.
+ * Copyright (c) 2010,2019 IBM Corporation and others.
```

Comment:
shouldn't update this since doesn't appear there is any need to update this file